addons/project/views/project_views.xml

```diff
@@ -1473,12 +1473,14 @@
                                             <span style="font-weight:bold;" class="fa fa-fw mt4 fa-meh-o text-warning" t-elif="record.rating_avg.raw_value &gt;= 2.33" title="Average Rating: Okay" role="img" aria-label="Neutral face"/>
                                             <span style="font-weight:bold;" class="fa fa-fw mt4 fa-frown-o text-danger" t-else="" title="Average Rating: Dissatisfied" role="img" aria-label="Sad face"/>
                                         </b>
+                                        <a t-if="record.subtask_count.raw_value and record.allow_subtasks.raw_value" class="subtask_list_button" title="See Subtasks" style="padding: 0px 5px 0px 0px; box-sizing: border-box; font-size: 20px;"/>
```

Comment:
Do we really need the style attribute? 🤔

Reply:
I used it because of the changes I make in the button's style attributes. I could pass the values from within the compiler but I figured that if in the future, someone wanted to change for example the font size, it would be easier to find it in the view.

Reply:
okay 🙂